README.md

```diff
@@ -2,7 +2,7 @@
 
 [Homebrew Formulae](https://formulae.brew.sh) is an online package browser for [Homebrew](https://brew.sh).
 
-It displays all packages in [Homebrew/homebrew-core](https://github.com/Homebrew/homebrew-core). It is deployed to GitHub Pages after every [Homebrew/homebrew-core](https://github.com/Homebrew/homebrew-core) commit by [Travis CI](https://travis-ci.org).
+It displays all packages in [Homebrew/homebrew-core](https://github.com/Homebrew/homebrew-core) and [Homebrew/homebrew-cask](https://github.com/Homebrew/homebrew-cask). It is deployed to GitHub Pages after every [Homebrew/homebrew-core](https://github.com/Homebrew/homebrew-core) and [Homebrew/homebrew-cask](https://github.com/Homebrew/homebrew-cask) commit by [Brew Test Bot](https://docs.brew.sh/Brew-Test-Bot).
```

Comment:
Thanks! Could this link to https://github.com/Homebrew/homebrew-core/blob/master/.github/main.workflow and https://github.com/Homebrew/homebrew-cask/blob/master/.github/main.workflow and state that GitHub Actions makes the commits?